pom.xml

```diff
@@ -31,8 +31,8 @@
 
     <properties>
         <inceptionYear>2022</inceptionYear>
-        <opentelemetry.spec.version>1.13.0</opentelemetry.spec.version>
-        <opentelemetry.java.version>1.18.0</opentelemetry.java.version>
+        <opentelemetry.spec.version>1.14.0</opentelemetry.spec.version>
```

Comment:
We need 1.13.0

Reply:
@brunobat I added that change now, including an update on the note, that points (fixed) to the final definition in otel-spec-version 1.14.0 for the SDK configuration property. I also added a comment to remember us to maintain this in a future release.